testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/transaction/ObjectStoreTypeTest.java

```diff
@@ -69,19 +69,19 @@ public void after() throws Exception {
     }
 
     @Test
-    public void testHornetQObjectStore() throws IOException, MgmtOperationException {
+    public void testJournalObjectStore() throws IOException, MgmtOperationException {
```

Comment:
We need a new test instead of repurposing the existing one. The use-hornetq-store attribute is still part of the management API so we should test what happens when it is used.